tools/assets_database/main.py

```diff
@@ -6,13 +6,9 @@
 from pathlib import Path
 from tempfile import TemporaryDirectory
 
-from gevent import monkey
```

Comment:
What's this for? How was this script called? The monkey patch here was, I assume for if this is called directly.

Reply:
It is called for example

```
python tools/assets_database/main.py --start-db 6 --target-version 20 --assets-branch  develop --target-directory data
```

I'm not sure what changed in develop but it was applying twice the patch and breaking requests

Reply:
How can that be?

The only place monkey patch is applied is `rotkehlchen/__main__.py

Which I don't think is ran here. So that means that by removing this, you are removing gevent monkey patching.

Or something else is off which we need to figure out.